### PR TITLE
Update WCF dependencies on the compat pack

### PR DESCRIPTION
**Pull request by safern**
We will update to the latest GA packages once they are ready.

Also, adding a new package which is a full façade to .NET Framework: `System.Web.Services.Description`.

I will also port this to 6.0

FYI: @mconnew @HongGit

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -12,6 +12,10 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>9f2345b3c5f43dbf34790e21657ae1f2445cd06a</Sha>
     </Dependency>
+    <Dependency Name="System.ServiceModel.Primitives" Version="4.9.0-rc2.21473.1">
```
Comment (ViktorHofer) [inline eng/Version.Details.xml:15]: @mmitche just checking, do we want this product dependency in the graph?

Comment (mmitche) [reply eng/Version.Details.xml:15]: Yep it's fine.

**Review by ericstj — APPROVED** [review]

**Review by HongGit — APPROVED** [review]

**Comment by safern** [comment]
Failures are unrelated and known, @jkoritzinsky is there an issue for the formatting job?

**Comment by jkoritzinsky** [comment]
Yes and there's already a PR in flight to fix the issue.

**Pull request merged**